<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QMainWindow" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
display: inline-block;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbDichvu_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>21</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>HCMS</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbEmail">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>SĐT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtEmail">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-radius: 10px; 
    border: 1px solid #cccccc;
    padding: 5px; 
    background-color: white; 
    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbPassword">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>MẬT KHẨU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtPassword">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-radius: 10px; /* Bo góc 10px */
    border: 1px solid #cccccc; /* Đường viền */
    padding: 5px; /* Khoảng cách bên trong */
    background-color: white; /* Nền trắng */
    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3);</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="bntSignIn">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">    border-radius: 10px; /* Bo góc 10px */
    border: 1px solid #cccccc; /* Đường viền */
    padding: 5px; /* Khoảng cách bên trong */
    background-color: white; /* Nền trắng */
    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3);
	background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>ĐĂNG NHẬP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
